pkg/bolttools/sandbox.go

```diff
@@ -0,0 +1,475 @@
+	}
+
+	namespaceOptions := linuxSandbox.GetNamespaceOptions()
+	if namespaceOptions == nil {
+		return fmt.Errorf("Linux sandbox config is missint Namespaces attribute: %#v", linuxSandbox)
+	}
+
+	err = b.db.Batch(func(tx *bolt.Tx) error {
+		parentBucket, err := tx.CreateBucketIfNotExists([]byte("sandbox"))
+		if err != nil {
+			return err
+		}
+
+		sandboxBucket, err := parentBucket.CreateBucketIfNotExists([]byte(podId))
+		if err != nil {
+			return err
+		}
+
+		if err := sandboxBucket.Put([]byte("createdAt"), []byte(strconv.FormatInt(time.Now().Unix(), 10))); err != nil {
+			return err
+		}
+
+		if err := sandboxBucket.Put([]byte("hostname"), []byte(config.GetHostname())); err != nil {
+			return err
+		}
+
+		if err := sandboxBucket.Put([]byte("logDirectory"), []byte(config.GetLogDirectory())); err != nil {
+			return err
+		}
+
+		if err := sandboxBucket.Put([]byte("labels"), []byte(strLabels)); err != nil {
+			return err
+		}
+
+		if err := sandboxBucket.Put([]byte("annotations"), []byte(strAnnotations)); err != nil {
+			return err
+		}
+
+		metadataBucket, err := sandboxBucket.CreateBucketIfNotExists([]byte("metadata"))
+		if err != nil {
+			return err
+		}
+
+		if err := metadataBucket.Put([]byte("name"), []byte(metadata.GetName())); err != nil {
+			return err
+		}
+
+		if err := metadataBucket.Put([]byte("uid"), []byte(metadata.GetUid())); err != nil {
+			return err
+		}
+
+		if err := metadataBucket.Put([]byte("namespace"), []byte(metadata.GetNamespace())); err != nil {
+			return err
+		}
+
+		if err := metadataBucket.Put([]byte("attempt"), []byte(strconv.FormatUint(uint64(metadata.GetAttempt()), 10))); err != nil {
+			return err
+		}
+
+		linuxSandboxBucket, err := sandboxBucket.CreateBucketIfNotExists([]byte("linuxSandbox"))
+		if err != nil {
+			return err
+		}
+
+		if err := linuxSandboxBucket.Put([]byte("cgroupParent"), []byte(linuxSandbox.GetCgroupParent())); err != nil {
+			return err
+		}
+
+		namespaceOptionsBucket, err := linuxSandboxBucket.CreateBucketIfNotExists([]byte("namespaceOptions"))
+		if err != nil {
+			return err
+		}
+
+		if err := namespaceOptionsBucket.Put([]byte("hostNetwork"), []byte(strconv.FormatBool(namespaceOptions.GetHostNetwork()))); err != nil {
+			return err
+		}
+
+		if err := namespaceOptionsBucket.Put([]byte("hostPid"), []byte(strconv.FormatBool(namespaceOptions.GetHostPid()))); err != nil {
+			return err
+		}
+
+		if err := namespaceOptionsBucket.Put([]byte("hostIpc"), []byte(strconv.FormatBool(namespaceOptions.GetHostIpc()))); err != nil {
+			return err
+		}
+
+		return nil
+	})
+
+	return err
+}
+
+func (b *BoltClient) RemovePodSandbox(podId string) error {
+	if err := b.db.Batch(func(tx *bolt.Tx) error {
+		bucket := tx.Bucket([]byte("sandbox"))
+		if bucket == nil {
+			return fmt.Errorf("Bucket 'sandbox' doesn't exist")
+		}
+
+		if err := bucket.DeleteBucket([]byte(podId)); err != nil {
+			return err
+		}
+
+		return nil
+	}); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
IMO easier would be replace this if with simple return `b.db.Batch(...)` output.